README.md

```diff
@@ -30,6 +30,22 @@ Currently we are targeting five platforms: OS X, Ubuntu Linux, iOS, Android, and
 
 ## platforms ##
 
+### Windows (command line) ###
+Downloading angle https://www.dropbox.com/s/vollhh52bjemeym/angle.7z?dl=0 and extract to `build` directory
```

Comment:
We can't direct users to download unverified DLLs from a 3rd party's Dropbox account. I don't have reason to mistrust this account or file in particular, but this just invites tons of potential problems (the files could execute malicious code, the files could be changed without our knowledge, the files could disappear entirely, etc.).